DNN Platform/DotNetNuke.Abstractions/Prompt/ICommandHelp.cs

```diff
@@ -1,33 +1,38 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
```

Comment:
Is this blank line required?  I had been removing this line.  I'm fine either way, just wondering if I was working against StyleCop when I was doing that.

Reply:
It is not a stylecop setting, for some reason I thought it was inside our stylecop configuration. I can remove these blank lines below the file header.

Reply:
I would prefer no white space, but happy to be overridden if there are any other opinions

Reply:
I have removed the whitespace, I'm going to leave this conversation open if anyone has strong opinions on this